### ISSUE TITLE
Some dependencies fail to compile!

### ISSUE THREAD

**Issue by VarLad** [Roger-luo/MinionsREPL.jl#7]
✗ PulseAudio_jll
  ✗ alsa_plugins_jll
  ✗ PortAudio

Just checking this out randomly :rofl:

**Comment by Roger-luo** [comment]
yeah I haven't got time to make this more serious ;-) so I deleted my post on discourse. I'm not sure why, what's your OS? I only tested this on Mac, it basically sings every time you type a letter in REPL

**Comment by VarLad** [comment]
Fedora Linux 35
Do I need to install something external to make it work? :rofl: 

Never mind, I had to install something called `libnsl` (fortunately available in the repos) to make the above dependencies compile
Sound still doesn't work though

**Comment by Roger-luo** [comment]
I think worth an issue in upstream packages PortAudio

**Comment by VarLad** [comment]
Just filed an issue
I wonder if its because Fedora ditched PulseAudio for PipeWire since an year now
